docs/releases.rst

```diff
@@ -1,6 +1,333 @@
 Release notes
 =============
 
+Mailu 2.0 - 2023-04-03
+----------------------
+
+Mailu 2.0 is finally available. It is vital to read the `Upgrading` section before upgrading to Mailu 2.0.
```

Comment:
Maybe add with "as it introduces major features and breaking changes from 1.9"

Reply:
83ce32f7